src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -51,7 +51,8 @@ public void setup() {
     Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/admin.jsp"))
         .thenReturn(mockRequestDispatcher);
 
-    UserStore mockUserStore = Mockito.mock(UserStore.class);
```

Comment:
Ya this line probably was not very helpful, probably your IDE should have flagged this as hiding an instance variable?

Reply:
I'm wondering why the above line --> UserStore mockUserStore = Mockito.mock(UserStore.class); <-- is incorrect
while the following line --> mockUserStore = Mockito.mock(UserStore.class); <-- is the correct way to format it. The first method is the way that I saw it in the CodeU guide.